.changelog/2604.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:bug
+resource/mongodbatlas_advanced_cluster: Supports using decimal in advanced_configuration `oplog_min_retention_hours`
+```
+
+```release-note:bug
+resource/mongodbatlas_cluster: Supports using decimal in advanced_configuration `oplog_min_retention_hours`
```

Comment:
I can't see any change in cluster, only adv_cluster

Reply:
cluster references the advanced config [here](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/CLOUDP-273994_oplog_min_retention_hours_allow_float/internal/service/cluster/resource_cluster.go#L312)

Reply:
oh, ok, thanks.  could we update cluster tests then (or create a new one) ?

Reply:
I don't think it is necessary. Both use SDK v2.

Reply:
*Turns out this test exposed another bug. Good to test 😅

Reply:
Will continue to work on that here
https://jira.mongodb.org/browse/CLOUDP-274185